### DISCUSSION TITLE
Facing issue while extracting text from a pdf with checkboxes

### DISCUSSION THREAD

**Discussion by kalyankumarp** [jsvine/pdfplumber#512] [Ask for help with specific PDFs]
Hi,

I am trying to extract text from a PDF that has check-boxes.  I am getting "l" in place of checkboxes with both extract_text and extract_words methods. Can anyone please tell me how to ignore checkboxes while extracting text/words? 

Thanks in advance.

**Comment by jsvine** [comment]
Hi, and thanks for your interest in this library. The best strategy will depend on how, exactly, the checkboxes are implemented in the PDF. Can you share the PDF in question?
